SwiftUIDZ/SwiftUIDZ/ClimateControlView/ClimateControlView.swift

```diff
@@ -0,0 +1,350 @@
+//
+//  ClimateControlView.swift
+//  SwiftUIDZ
+//
+//  Created by Алена Панченко on 31.01.2023.
+//
+
+import SwiftUI
+
+/// Вью - модель настроек системы кондиционирования
+final class ClimateControlViewModel: ObservableObject {
+    
+    @Published var gradientColors = [Color("GradientBackgroundTop"), Color("GradientBackGroundBottom")]
+    @Published var settingsIndex = 0
+ 
+   
+    @Published var settingsClimate = SettingsClimateModel.getSettings()
+    
+    @Published var climateSliderValues = [0.0, 0.0, 0.0, 0.0]
+    
+    @Published var isClimateControlEnabled = false
+    @Published var bottomSheetCurrentOffsetY: CGFloat = 0
+    @Published var bottomSheetLastOffsetY: CGFloat = 0
+    @Published var bottomGradientColor: CGColor = CGColor(red: 47, green: 184, blue: 255, alpha: 1)
+    
+  
+    @Published var topGradientColor: CGColor = CGColor(red: 158, green: 236, blue: 217, alpha: 1)
+    
+    // MARK: - Public Methods
+    
+//    func getSettingsName() -> String {
```

Comment:
```suggestion
//    func getSettingsName() -> String {
```
нельзя оставлять так